### PR TITLE
use cases implemented

### PR DESCRIPTION
**Pull request by Matias-Villalobo**
In this commit i have done the use-cases for the getCharacters. 

image

### CODE REVIEW

File: app/src/main/java/com/example/marvelapp/presentation/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import com.example.marvelapp.presentation.mvp.presenter.MarvelAppPresenter
+import com.example.marvelapp.presentation.mvp.view.MarvelAppView
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var presenter: MarvelAppContract.MarvelAppPresenter
+    private lateinit var binding: ActivityMainBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        val getCharacterUseCase = GetCharactersUseCaseImpl(MarvelServiceImpl())
```
Comment (SartiMau) [inline app/src/main/java/com/example/marvelapp/presentation/MainActivity.kt]: you can unify this line with the line 24

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/marvelapp/data/serviceImpl/MarvelServiceImpl.kt [hunk 2]
```diff
@@ -1,9 +1,10 @@
-package com.example.marvelapp.data.service
+package com.example.marvelapp.data.serviceImpl
```
Comment (SartiMau) [inline app/src/main/java/com/example/marvelapp/data/serviceImpl/MarvelServiceImpl.kt]: you should let this package as service

File: app/src/main/java/com/example/marvelapp/domain/usecases/GetCharactersUseCaseImpl.kt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.example.marvelapp.domain.usecases
```
Comment (SartiMau) [inline app/src/main/java/com/example/marvelapp/domain/usecases/GetCharactersUseCaseImpl.kt]: this package should be usecase no usecases

**Review by SartiMau — CHANGES_REQUESTED** [review]

**Review by LuchoElissondoGlobant — APPROVED** [review]

File: app/src/main/java/com/example/marvelapp/domain/usecases/GetCharactersUseCaseImpl.kt (hunk 3 above)
Comment (Matias-Villalobo) [reply app/src/main/java/com/example/marvelapp/domain/usecases/GetCharactersUseCaseImpl.kt]: Done thx

File: app/src/main/java/com/example/marvelapp/presentation/MainActivity.kt (hunk 1 above)
Comment (Matias-Villalobo) [reply app/src/main/java/com/example/marvelapp/presentation/MainActivity.kt]: Done thx

File: app/src/main/java/com/example/marvelapp/data/serviceImpl/MarvelServiceImpl.kt (hunk 2 above)
Comment (Matias-Villalobo) [reply app/src/main/java/com/example/marvelapp/data/serviceImpl/MarvelServiceImpl.kt]: Done thx now its call "implementation"

**Review by SartiMau — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Pull request merged**